promptxchange-frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
there's a general .gitignore file in the root directory, please copy these to the bottom of that file. You can add a comment like:

'# FRONT END PATTERNS TO IGNORE

...

Reply:
Done